### PR TITLE
Add SpringBootAdmin Docker image

### PR DESCRIPTION
**Pull request by hananiel**

### CODE REVIEW

File: spring-boot-admin/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# -----------------------------------------------------------------------------
+# Spring Boot Admin Server build
+# -----------------------------------------------------------------------------
+
+FROM openjdk:8u171-jdk-alpine as build
```
Comment (ccheetham) [inline spring-boot-admin/Dockerfile:5]: We've migrated to using Liberica JDKs.

Sample image: `bellsoft/liberica-openjdk-alpine:8`

File: spring-boot-admin/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+WORKDIR /scratch
+COPY files .
+RUN ./gradlew build
+RUN ln -s build/libs/springbootadmin-${IMAGE_VERSION}.jar springbootadmin.jar
+
+# -----------------------------------------------------------------------------
+# Spring Boot Admin Server image
+# -----------------------------------------------------------------------------
+
+FROM openjdk:8u171-jre-alpine
```
Comment (ccheetham) [inline spring-boot-admin/Dockerfile:17]: see above note

File: spring-boot-admin/files/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,5 @@
+FROM openjdk:8-jdk-alpine
```
Comment (ccheetham) [inline spring-boot-admin/files/Dockerfile:1]: Do we need this file?

File: spring-boot-admin/files/gradlew [hunk 4]
```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env sh
```
Comment (ccheetham) [inline spring-boot-admin/files/gradlew:1]: This file needs to be executable (755)

File: spring-boot-admin/manifest.yml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+image: steeltoeoss/spring-boot-admin:2.2.3
+tags: ['latest', '2.2.3']
+manifests:
+  - image: steeltoeoss/mysql-amd64-linux:5.7.24
```
Comment (ccheetham) [inline spring-boot-admin/manifest.yml:4]: `image` should be: `steeltoeoss/spring-boot-admin-amd64-linux:2.2.3`

File: spring-boot-admin/README.adoc [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+:toclevels: 1
+:!toc-title:
+:linkattrs:
+
+SteeltoeOSS https://github.com/codecentric/spring-boot-admin docker image.
+
+== Running
+
+----
+$ docker run --publish 8080:8080 steeltoeoss/spring-boot-admin:latest
```
Comment (ccheetham) [inline spring-boot-admin/README.adoc:12]: `:latest` seems superfluous here

File: spring-boot-admin/files/gradlew (hunk 4 above)
Comment (ccheetham) [reply spring-boot-admin/files/gradlew:1]: If not trivial to do from a Windows box, we can merge as is and fix afterwards.

File: spring-boot-admin/files/Dockerfile (hunk 3 above)
Comment (hananiel) [reply spring-boot-admin/files/Dockerfile:1]: Not sure I understand this comment? you mean the line (for multi stage docker file or what)? results in a smaller image.

File: spring-boot-admin/Dockerfile (hunk 1 above)
Comment (hananiel) [reply spring-boot-admin/Dockerfile]: done.

File: spring-boot-admin/Dockerfile (hunk 2 above)
Comment (hananiel) [reply spring-boot-admin/Dockerfile]: done.

File: spring-boot-admin/manifest.yml (hunk 5 above)
Comment (hananiel) [reply spring-boot-admin/manifest.yml]: fixed.

File: spring-boot-admin/README.adoc (hunk 6 above)
Comment (hananiel) [reply spring-boot-admin/README.adoc]: removed.

File: spring-boot-admin/files/gradlew (hunk 4 above)
Comment (hananiel) [reply spring-boot-admin/files/gradlew:1]: fixed from a mac.

File: spring-boot-admin/files/Dockerfile (hunk 3 above)
Comment (ccheetham) [reply spring-boot-admin/files/Dockerfile:1]: There's a Dockerfile in the parent dir that is one you crafted.  What's the purpose of this one?

**Pull request merged**